### DISCUSSION TITLE
window[arg.funcName] is not a function

### DISCUSSION THREAD

**Discussion by winst0niuss** [getgauge/taiko#2600] [General]
I got this error, why?

**Comment by liolick** [comment]
It seems something changed in last Chrome release.
Now this internal Taiko error reaches global window.onerror handler.
If application has such a handler and show errors to user it will show this internal error too.

**Reply by zabil** [reply]
Do you have a sample script to reproduce this error?

**Reply by winst0niuss** [reply]
> It seems something changed in last Chrome release. Now this internal Taiko error reaches global window.onerror handler. If application has such a handler and show errors to user it will show this internal error too.

Is it possible to close this window with any command?

**Reply by liolick** [reply]
@zabil Yes
Test page on Sandbox: https://codesandbox.io/s/javascript-forked-nr4b4y?file=/index.js
Taiko test script:

(async () => {
    try {
        await openBrowser();
        await goto("https://nr4b4y.csb.app/");
        await waitFor("test", 20000);
    } catch (error) {
        console.error(error);
    } finally {
        await closeBrowser();
    }
})();

Taiko must be run in observe mode!
Error does not reproduce in silent mode.

**Reply by liolick** [reply]
@Smitcore Depends on window. How it realised.
